docs/generate_genome_attributes.md

```diff
@@ -0,0 +1,24 @@
+# Generate Genome Attributes
+
+This document describes the steps and scripts used to generate attributes of a genome. 
+The scripts used in the process are located in the `src/loaders` directory.
+
+1. Retrieve Source Genome
+   * The first step is to download the genome files from the NCBI FTP server using the script 
```

Comment:
Shortly there will be other options for getting data, can you make this a bit more general for when those cases are checked into the repo (e.g. workspace data)

Reply:
👍

Reply:
:+1: